corehq/apps/api/resources/v0_4.py

```diff
@@ -161,7 +161,7 @@ class Meta(CustomResourceMeta):
         list_allowed_methods = ['get']
         detail_allowed_methods = ['get']
         resource_name = 'form'
-        ordering = ['received_on', 'server_modified_on']
+        ordering = ['received_on', 'server_modified_on', 'indexed_on']
```

Comment:
out of curiosity, where does `indexed_on` come from? it isn't in the ES mappings, but maybe I'm misunderstanding exactly what this does.

Reply:
In ES it's called `inserted_at`:
https://github.com/dimagi/commcare-hq/blob/5ab2ab87bdf4b77f008cb093a5f934c69ccbf4fb/corehq/pillows/case.py#L39
https://github.com/dimagi/commcare-hq/blob/2457bd0035c3be3b47138ef96bcdfcd2750a9c65/corehq/pillows/mappings/case_mapping.py#L55